### ISSUE TITLE
Create conda package?

### ISSUE THREAD

**Issue by IAlibay** [MDAnalysis/hole2#2]
Related to https://github.com/MDAnalysis/mdanalysis/pull/3490#discussion_r779460269

It probably would be easier if we could create a conda package for hole2.

**Comment by lilyminium** [comment]
We need to tag or make a release for a recipe, I think -- what would be the best thing to call it? v2.2.005b?

**Comment by IAlibay** [comment]
@orbeckst last I remembered you tried to get hold of @osmart to fix: https://github.com/osmart/hole2/pull/2 ?

Do we have any idea if we'd be ok to make the conda-forge package (probably for this fork?)

**Comment by orbeckst** [comment]
I assume we’d be ok to make a conda package. We can always make @osmart a maintainer if he wants to.

**Comment by ianmkenney** [comment]
@orbeckst @IAlibay @lilyminium I'm currently working on a [conda-forge package](https://github.com/conda-forge/staged-recipes/pull/21800). I hadn't realized the latest tag was from 2016—we definitely need a new one. The `Makefile` from that point doesn't have an install target. Additionally, #3 brings up a good point and should be considered.

**Comment by IAlibay** [comment]
@ianmkenney - yes that sounds good. Are you ok reviewing #3 and starting the process for a release? (unless @orbeckst objects I'm happy to hand over the super powers to do that)

**Comment by orbeckst** [comment]
no objections

**Comment by IAlibay** [comment]
> no objections

@lilyminium approves too - @ianmkenney is now a maintainer for this repo.

**Comment by osmart** [comment]
Hello everyone

Apologies for not having time to spend on hole2 maintenance. If you want to continue working on the fork then I am happy. If instead, you would like to become collaborators on the original repo then let me know and I will do this.

Best regards,

Oliver

**Comment by orbeckst** [comment]
Hello @osmart !

We would be happy to work on your original repository and retire our fork here. Our goal would be to keep HOLE available as a mature program and only make changes to keep it running as is. An immediate objective is to create a conda-forge package to make it widely available to anyone using conda. 

Our main concern is that if there is only a limited number of code reviewers available, development slows down too much and we cannot progress. Thus our two main questions are:

* We are not experts on HOLE itself. Are you able to respond to questions regarding HOLE's core in the future, if necessary?
* We can modernize the build system and we have some patches that we would want to merge (that's the main reason that _this_ fork exists). We would like to be able to progress on such peripheral changes quickly using our usual PR/review/merge approach. Realistically, we would need to be able to act on these quickly without waiting several weeks for code reviews. Therefore, would you be willing to grant write access (and thus the ability to review and merge PRs) to a number of MDAnalysis developers?

In summary, we'd be happy to help maintain HOLE but we'd like to hear from you how you envisage this to work.

— Oliver (B) @orbeckst (on behalf of the @mdanalysis/coredevs)

**Comment by osmart** [comment]
Hi Oliver (B),

I am glad to see that HOLE is still useful after 30 years and I want to support its continued use.

I have a limited amount of time but promise to respond to any questions promptly. 

A conda build seems a great idea. 

I am happy to grant access to a number of MDAnalysis developers.


- Oliver (S)

**Comment by osmart** [comment]
@orbeckst I have sent you a collaboration invitation - is this the best way to do things? (I use GitLab for my current work and things are slightly different there).

**Comment by orbeckst** [comment]
@osmart , thank you for adding me. I can approve/merge PRs and create releases if necessary.

Could you please add the following MDAnalysis folks as collaborators on https://github.com/osmart/hole2  as well?
- @ianmkenney 
- @lilyminium 
- @IAlibay 

Everyone on this list is involved in HOLE-related work and/or releases so this will allow us to move relatively quickly. In particular, it removes _me_ as a bottleneck.

If you haven't done so already, I'd suggest you set the branch protection for the master branch to require a review: Settings -> Branches:  Branch protection rules: 
- pattern: master
- check: require a pull request before merging
  - check: require approval (1)   

Just as a safe-guard so that you (and anybody else) has a chance to review any changes.

Thanks!

**Comment by osmart** [comment]
- [x]  I have invited @ianmkenney @lilyminium @IAlibay
- [x] Have added the  Settings -> Branches: Branch protection rules you suggested:

>  ![Screenshot 2023-01-24 at 18 34 00](https://user-images.githubusercontent.com/8986794/214379036-50227f12-043c-4eea-b953-caccabd8887f.png)

**Comment by IAlibay** [comment]
Thanks @osmart !

**Comment by ianmkenney** [comment]
@osmart much appreciated!

**Comment by orbeckst** [comment]
With https://github.com/osmart/hole2/issues/6 closed we now have a conda package and we can retire the fork #4 (after making sure that MDA uses the cf package https://github.com/MDAnalysis/mdanalysis/issues/4012 ).
